main/CommandHandler.cpp

```diff
@@ -2489,26 +2488,26 @@ void CommandHandlerClass::updateGpio0Pin()
 {
   xSemaphoreTake(_updateGpio0PinSemaphore, portMAX_DELAY);
 
-  bool available = false;
+  int available = 0;
```

Comment:
this is not ideal, but make the change to keep it as close as stock/upstream arduino as possible.

Reply:
No problem changing it back. I think I changed it to make it more readable, but I agree it makes diff-ing harder.